app/controllers/bookable_slots_controller.rb

```diff
@@ -21,6 +19,15 @@ def scoped_to_me?
 
   private
 
+  def bookable_slots
```

Comment:
I'm starting to think this should be a scope on `BookableSlot`, or at least renamed to something more obvious like `visible_bookable_slots`, which would explain why we aren't removing the slots obscured appointments.

As it is, these really aren't `bookable_slots` at all...